apps/cli/src/handleLoginCliCommand.ts

```diff
@@ -7,27 +8,54 @@ import { boldText, colorizeText, openURL } from "./utils.js";
 
 const ACCESS_TOKEN_REQUESTED_BY_CLI_KEY = "accessTokenRequestedByCLI";
 
+const routeUserToStudioForLogin = (printer: PrinterBlueprint) => {
+	printer.printConsoleMessage(
+		"info",
+		colorizeText(
+			"Opening the Codemod Studio... Please Sign in there!\n",
```

Comment:
Avoid studio. We need to change that soon. 

"Redirecting to Codemod sign-in page..."

@hbjORbj